org/org.yaml

```diff
@@ -105,3 +105,156 @@ orgs:
     - withlin
     - wlynch
     - yuege01
+    teams:
+      catalog.maintainers:
+        description: "the catalog maintainers"
+        maintainers:
+        - vdemeester
+        - ImJasonH
+        - bobcatfish
+        members:
+        - kimsterv
+        - dlorenc
+        - sbwsg
+        - chmouel
+        privacy: closed
```

Comment:
as this is publically available here, we could make it public but :stuck_out_tongue_closed_eyes: